src/main/resources/config/application-prod.yml

```diff
@@ -44,16 +50,21 @@ spring:
         prepStmtCacheSize: 250
         prepStmtCacheSqlLimit: 2048
         useServerPrepStmts: true
+        socketFactory: com.google.cloud.sql.mysql.SocketFactory
```

Comment:
same as above, production profile properties shouldn't be tied with `gcp`

Reply:
Thanks. Done. :+1: